internal/cephfs/volume.go

```diff
@@ -42,11 +42,13 @@ const (
 type Subvolume struct {
 	BytesQuota    int      `json:"bytes_quota"`
 	DataPool      string   `json:"data_pool"`
+	Features      []string `json:"features"`
 	GID           int      `json:"gid"`
 	Mode          int      `json:"mode"`
 	MonAddrs      []string `json:"mon_addrs"`
 	Path          string   `json:"path"`
 	PoolNamespace string   `json:"pool_namespace"`
+	State         string   `json:"state"`
```

Comment:
@Yuggupta27 please feel free to add a comment about this field and its optional values. so that it will be useful later.

Reply:
Sure Humble, will add it.